### PR TITLE
[Python] Bubble Sort

### PR DESCRIPTION
**Pull request by cdefga**

### CODE REVIEW

File: Python/Algorithms/bubble_sort.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+def bubble_sort(array):
```
Comment (amanmehara) [inline Python/Algorithms/bubble_sort.py:1]: Please add the license header.

**Review by amanmehara — CHANGES_REQUESTED** [review]
Please have a look at [CONTRIBUTING.md](https://github.com/amanmehara/programming/blob/master/CONTRIBUTING.md) and make changes accordingly.
I'll approve and merge afterwards! 😃

**Comment by cdefga** [comment]
I changed

**Review by amanmehara — APPROVED** [review]
Thanks for contributing! 🎉

**Pull request merged**